evap/grades/templates/grades_semester_view.html

```diff
@@ -11,78 +11,79 @@
         {{ semester.name }}
     </h3>
 
-    <div class="panel panel-default">
-        <div class="panel-heading">
-            <h3 class="panel-title">{% trans "Courses" %}</h3>
+    <div class="card">
+        <div class="card-header">
+            {% trans "Courses" %}
         </div>
-        <div class="panel-body">
+        <div class="card-body">
             {% if courses %}
                 <table class="table table-striped grade-course-table vertically-aligned">
                     <thead>
                         <tr>
-                            <th class="col-sm-5">{% trans "Name" %}</th>
-                            <th class="col-sm-3">{% trans "Responsible" %}</th>
-                            <th class="col-sm-1">{% trans "Evaluation completed" %}</th>
-                            <th class="col-sm-1">{% trans "Midterm grade documents" %}</th>
-                            <th class="col-sm-1">{% trans "Final grade documents" %}</th>
-                            <th class="col-sm-1 text-right" data-orderable="false">{% trans "Actions" %}</th>
+                            <th style="width: 35%">{% trans "Name" %}</th>
+                            <th style="width: 20%">{% trans "Responsible" %}</th>
+                            <th style="width: 10%">{% trans "Evaluation completed" %}</th>
+                            <th style="width: 10%">{% trans "Midterm grade documents" %}</th>
+                            <th style="width: 10%">{% trans "Final grade documents" %}</th>
+                            <th style="width: 15%" class="text-right" data-orderable="false">{% trans "Actions" %}</th>
                         </tr>
                     </thead>
                     <tbody>
                         {% for course, num_midterm_grades, num_final_grades in courses %}
                             <tr>
                                 <td>
-                                    <a href="{% url "grades:course_view" semester.id course.id %}" {{ disable_if_archived }}>{{ course.name }}</a><br/>
+                                    {% if semester.is_archived %}
+                                        {{ course.name }}
+                                    {% else %}
+                                        <a href="{% url "grades:course_view" semester.id course.id %}">{{ course.name }}</a>
+                                    {% endif %}
```

Comment:
see below

Reply:
semester.is_archived is in an if around that part, so that's fine